entity/entity.proto

```diff
@@ -51,17 +49,20 @@ message BankPaymentMethod {
   string holder_name = 3;
   string bank_key = 4;
   string bank_account_number = 5;
-  string currency = 6;
+  string supported_currency = 6;
```

Comment:
Why `supported_currency` vs `currency` ? Looks like it refers to the same concept.

Reply:
renamed to supported_currency because that is also on the cryptopaymentmethod.
not super important which name we use as long as it is the same on both